Corona-Warn-App/src/main/java/de/rki/coronawarnapp/eventregistration/TraceLocationSettings.kt

```diff
@@ -0,0 +1,23 @@
+package de.rki.coronawarnapp.eventregistration
+
+import de.rki.coronawarnapp.eventregistration.storage.TraceLocationPreferences
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class TraceLocationSettings @Inject constructor(val preferences: TraceLocationPreferences) {
```

Comment:
what is the difference between `TraceLocationSettings` and `TraceLocationPreferences `? 
Can they be one class `TraceLocationSettings` ?

Reply:
i can do it in one class, if you want. generally i like to keep storage (preferences) from domain logic (settings) but maybe i am splitting hairs here....

Reply:
changed it